website/data/docs-nav-data.json

```diff
@@ -615,6 +615,10 @@
         "title": "Requirements",
         "path": "ecs/requirements"
       },
+      {
+        "title": "Resource Usage",
+        "path": "ecs/resource-usage"
```

Comment:
What do you think about instead adding this as a "Task Resource Usage" section in the Requirements page?

I'm good with it either way.